lightbug_http/http/http_version.mojo

```diff
@@ -0,0 +1,25 @@
+# TODO: Can't be used yet.
+# This doesn't work right because of float point
+# Shenaningans and round() doesn't give me what I want
+@value
+@register_passable("trivial")
```

Comment:
I think the float logic is not necessary, since we're not going to support 0.9 or 1.0 . It's always just going to be 1.1, which we can just call 1 in my opinion, , 2 and 3. 
So this could just be three if statements e.g if string is "HTTP/1.1" then version is 1, and so on. And we can mention in a comment that this is our convention for 1.1